Banking/src/main/java/com/example/Banking/service/TransactionService.java

```diff
@@ -0,0 +1,37 @@
+package com.example.Banking.service;
+
+import com.example.Banking.model.Account;
+import com.example.Banking.model.Transaction;
+import org.springframework.beans.factory.annotation.Autowired;
+
+
+@org.springframework.stereotype.Service
+public class TransactionService {
+
+    @Autowired
+    private TransactionDao transactionDao;
+    @Autowired
+    private AccountDao accountDao;
+
+
+    public String transaction(Transaction sender) throws Exception {
```

Comment:
Variable names should show the purpose they are serving. What does sender represent? It should be transaction.